azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/redux/action/CallingAction.kt

```diff
@@ -11,7 +11,10 @@ internal sealed class CallingAction : Action {
     class CallEndRequested : CallingAction()
     class HoldRequested : CallingAction()
     class ResumeRequested : CallingAction()
-    class StateUpdated(val callingState: CallingStatus) : CallingAction()
+    class StateUpdated(
```

Comment:
Naming.

CallingStatusUpdated : CallingAction

Reply:
As per discussion we will fix for GA.